CODE_OF_CONDUCT.md

```diff
@@ -1,128 +1,51 @@
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned to this Code of Conduct, and will communicate reasons for moderation
-decisions when appropriate.
-
-## Scope
-
-This Code of Conduct applies within all community spaces, and also applies when
-an individual is officially representing the community in public spaces.
-Examples of representing our community include using an official e-mail address,
-posting via an official social media account, or acting as an appointed
-representative at an online or offline event.
+# Catppuccin's Community Guidelines
+
+This document defines guidelines for any person(s) participating in the
+Catppuccin community, whether they are a contributor, user, or observer. We
+want to ensure that everyone feels welcome, safe and included in our community.
+
+The guidelines apply to all "collaboration platforms" used by the Catppuccin
+community, which can be defined as any platform where members interact with
+each other. This includes, but is not limited to, GitHub, Gitea, Discord and
+any other platforms that may be used in the future.
+
+1. **Respect Diverse Perspectives**
+  - Participants are expected to be tolerant of opposing views, disagreements
+  should be addressed with respect and courtesy.
+2. **Civil Communication**
+  - Participants should communicate professionally and constructively, focusing
+  on the merits of ideas rather than targeting individuals.
+3. **Assumption Of Good Intentions**
+  - Participants should always assume good intentions. Misunderstandings may
+  occur, and it is important to give others the benefit of the doubt before
+  jumping to conclusions.
+4. **Zero Tolerance Of Harassment**
+  - Harassment, including offensive comments, unwanted advances, intimidation,
+  or any behavior creating discomfort, will not be tolerated.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-core@catppuccin.com.
-All complaints will be reviewed and investigated promptly and fairly.
+> [!IMPORTANT]  
+> Instances of abusive, harassing, or otherwise unacceptable
+> behavior may be reported to the community leaders responsible for enforcement
+> at [core@catppuccin.com](mailto:core@catppuccin.com). All complaints will be
+> reviewed and investigated promptly and fairly.
+>
+> All community leaders are obligated to respect the privacy and security of
+> the reporter of any incident.
 
-All community leaders are obligated to respect the privacy and security of the
-reporter of any incident.
+TODO: Explain why enforcement guidelines like GitHub/Gitea are separate from
+platforms like Discord.
 
-## Enforcement Guidelines
+### GitHub / Gitea
 
-Community leaders will follow these Community Impact Guidelines in determining
-the consequences for any action they deem in violation of this Code of Conduct:
+TODO
 
-### 1. Correction
+### Discord
```

Comment:
Currently not a fan of how I'm specifying specific platforms, would prefer to have generalised terms that apply to platforms like Discord/Twitter vs GitHub/Gitea